Sources/Document/Util/YorkieJSONObjectable.swift

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2022 The Yorkie Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+/// ``YorkieJSONObjectable`` provide a way to make a dictionary including members of a type confirming ``YorkieJSONObjectable``.
+public protocol YorkieJSONObjectable {
+    /// The members of ``excludedLabels`` is not included in a dictionary made by ``toYorkieObject``.
+    var excludedLabels: [String] { get }
+}
+
+public extension YorkieJSONObjectable {
+    /// ``toJsonObject`` make a dictionary including members of a type confirming ``YorkieObjectable``.
+    internal var toJsonObject: [String: Any] {
+        var result = [String: Any]()
+        Mirror(reflecting: self).children.forEach { child in
```

Comment:
How about applying Codable?

Reply:
I have changed ``YorkieJSONObjectable`` from ``Mirrot`` to Codable.